cmd/kubeadm/app/cmd/phases/bootstraptoken.go

```diff
@@ -61,7 +61,7 @@ func NewBootstrapTokenPhase() workflow.Phase {
 	return workflow.Phase{
 		Name:    "bootstrap-token",
 		Aliases: []string{"bootstraptoken"},
-		Short:   bootstrapTokenLongDesc,
+		Short:   "Generates bootstrap tokens used to join a node to a cluster",
```

Comment:
i think this makes the var unused.

Reply:
The var is used in the long description still.

Reply:
fixing.